packages/modals/src/utils/useModalContext.ts

```diff
@@ -9,9 +9,11 @@ import { createContext, useContext, HTMLAttributes } from 'react';
 
 export interface IModalContext {
   isLarge?: boolean;
+  isCloseButtonPresent?: boolean;
   getTitleProps: <T>(options?: T) => T & HTMLAttributes<HTMLDivElement>;
   getContentProps: <T>(options?: T) => T & HTMLAttributes<HTMLDivElement>;
   getCloseProps: <T>(options?: T) => T & HTMLAttributes<HTMLButtonElement>;
+  setCloseButtonPresent: (isPresent: boolean) => void;
```

Comment:
Not requesting a change since I'm not sure how I'd refactor for the current approach.

Seems like the newly added values are fit nicely in this interface. Context interfaces represent a blueprint of the core/essential methods and properties that is fundamental to the component API. The newly added values are relating to implementation details.

Reply:
@hzhu this one is a bit tricky for me to understand 😅 this is an internal interface and I wasn't sure but is there a standard for that? Isn't this more open-ended or are there things to consider with this?